controllers/controllers.js

```diff
@@ -76,3 +77,16 @@ exports.patchArticle = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  removeComment(comment_id)
+    .then((comment) => {
+      if (comment.rowCount === 0) {
+        res.status(404).send({ msg: 'not found' })
```

Comment:
Should be handling this in your custom error handler ideally - but up to you if you want to be consistent with the rest of your app